### PR TITLE
1385000 sorting filtering and paging ef

### PR DESCRIPTION
**Pull request by john-par**
Update for task to use .NET tutorial template.

### CODE REVIEW

**Comment by john-par** [comment]
@tdykstra this PR is ready for your review. Please let me know what you'd like changed.

Also, the metadata title (_and thus the H1 title_) exceeds the [markdown template](https://review.docs.microsoft.com/en-us/help/contribute/templates/dotnet/tutorial-template-dotnet?branch=master#markdown-template) limit of "115-145 characters including spaces." I was not sure how to shorten the title without loosing value. If you'd like the title to fit within the 115-145 character limit, can you please let me know what a good alternative is.

**Comment by Rick-Anderson** [comment]
@john-par 	which title? I shortened the following and it's now 78

**Add sorting, filtering, and paging with Entity Framework in an ASP.NET MVC app**

**Comment by john-par** [comment]
@Rick-Anderson you addressed the titles I was referring to. I didn't know that we could remove the "| Microsoft Docs" and introduce abbreviations in the meta title and H1 title.  Thank you.

The build is throwing a warning. It may be because the meta title is no longer in quotes. Shall I replace the quotes, or wait until @tdykstra has had a chance to provide the review you requested?

**Comment by Rick-Anderson** [comment]
Here's the error
(Line: 2, Col: 16, Idx: 168) - (Line: 2, Col: 16, Idx: 168): Mapping values are not allowed in this context.

We never use "double quotes". I added them to see if that fixes the problem. It might not like `:`

**Comment by john-par** [comment]
Thank you @Rick-Anderson. That fixed the warning. If you'd like me to change anything else in the article let me know. Otherwise, this PR is ready to be merged.

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/sorting-filtering-and-paging-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 1]
```diff
@@ -1,29 +1,37 @@
 ---
 uid: mvc/overview/getting-started/getting-started-with-ef-using-mvc/sorting-filtering-and-paging-with-the-entity-framework-in-an-asp-net-mvc-application
-title: "Sorting, Filtering, and Paging with the Entity Framework in an ASP.NET MVC Application | Microsoft Docs"
+title: "Tutorial: Add sorting, filtering, and paging with the EF in an ASP.NET MVC app"
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/sorting-filtering-and-paging-with-the-entity-framework-in-an-asp-net-mvc-application.md]:
```suggestion
title: "Tutorial: Add sorting, filtering, and paging with EF in an ASP.NET MVC app"
```

**Comment by john-par** [comment]
One suggestion committed. This PR is ready to be merged.

**Comment by john-par** [comment]
Trying to add the **Get the code** section to this PR and my local branch is corrupted. I am closing this PR. I will open a new PR based on a clean branch and resubmit.

**Pull request closed without merging**